### ISSUE TITLE
GPU usage is 100% even when switched back to original look

### ISSUE THREAD

**Issue by JusticeBeaver5** [sultim-t/xash-rt#177]
Not sure if that's an "issue" but it looks like all ray tracing still computed even when I click X to switch back to the old look. GPU utilization is 100%

**Comment by mastan-rus** [comment]
I think it only changes lighting of textures, uses original lightmaps instead of RT global illumination. All reflective surfaces still reflect everything so RT still works.

**Comment by JusticeBeaver5** [comment]
Yeah makes sence now. It says "Press X to switch renderers on the fly" in the description but doesnt specify which renderers so I assumed the old one and rtx.
